contribs/gnodev/cmd/gnobro/main.go

```diff
@@ -0,0 +1,452 @@
+package main
```

Comment:
Having autocomplete for realm searches would be *sick*. I'd also love to see package source code being rendered, not just stuff that has a `Render` method